packages/api/src/tasks/checkAlerts.ts

```diff
@@ -127,54 +210,80 @@ const buildEventSlackMessage = ({
 
 const fireChannelEvent = async ({
   alert,
-  logView,
-  totalCount,
+  dashboard,
+  endTime,
   group,
+  logView,
   startTime,
-  endTime,
+  totalCount,
+  windowSizeInMins,
 }: {
-  alert: IAlert;
-  logView: Awaited<ReturnType<typeof getLogViewEnhanced>>;
-  totalCount: number;
+  alert: AlertDocument;
+  logView: Awaited<ReturnType<typeof getLogViewEnhanced>> | null;
+  dashboard: {
```

Comment:
curious why build a one-off as opposed to being consistent with the dashboard schema?

Reply:
being lazy messing around the populated dashboard type. fixed it!